src/store/myPosts/actions.js

```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
```

Comment:
i saw in another file you called it myAxios, keep consistency

Reply:
Yes!! will keep consistency